### PR TITLE
SONAR-7565 add RESTARTING status in /api/system/status

### PR DESCRIPTION
**Pull request by sns-seb**
based on branch 5.5

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/platform/Platform.java [hunk 1]
```diff
@@ -157,10 +157,12 @@ public Status status() {
...
       return Status.BOOTING;
     }
-    if (levelSafeMode != null && currentLevel == levelSafeMode) {
+    PlatformLevel current = this.currentLevel;
+    PlatformLevel levelSafe = this.levelSafeMode;
+    if (levelSafe != null && current == levelSafe) {
       return Status.SAFEMODE;
     }
-    if (currentLevel == level4) {
+    if (current == level4) {
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/platform/Platform.java]: MAJOR Change this comparison to use the equals method. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1698)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* MAJOR 2 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [StatusAction.java#L93](https://github.com/SonarSource/sonarqube/blob/7da8c5b666732d3789c1a2d8023f7976ab0b5b4d/server/sonar-server/src/main/java/org/sonar/server/platform/ws/StatusAction.java#L93): The Cyclomatic Complexity of this method "computeStatus" is 11 which is greater than 10 authorized. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)

**Comment by julienlancelot** [comment]
LGTM !

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* MAJOR 2 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [StatusAction.java#L93](https://github.com/SonarSource/sonarqube/blob/8f2951628d59bc8e46b720373258ae932980c8ec/server/sonar-server/src/main/java/org/sonar/server/platform/ws/StatusAction.java#L93): The Cyclomatic Complexity of this method "computeStatus" is 11 which is greater than 10 authorized. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)

**Pull request merged**